lte/gateway/c/core/oai/include/s8_messages_types.h

```diff
@@ -42,6 +42,13 @@ typedef struct s8_create_session_response_s {
   fteid_t pgw_s8_cp_teid;
   uint32_t cause;
   protocol_configuration_options_t pco;
+  uint32_t temporary_session_id;  // temporary_session_id is created per pdn
+                                  // session while sending create session
+                                  // request and is temporarily used to fetch
+                                  // session, once sgw_s8_teid is allocated by
+                                  // orc8r, temporary session id is removed and
+                                  // shall update imsi_ue_context_htbl based on
+                                  // received sgw_s8_teid
```

Comment:
just curious, but is this field related with a session? Meaning, since we replaced teid. wouldn't it be better to use something like `temporary_teid` or `temporary_create_session_request_identifier`

Just saying because session_id reminds me to the id provided by sessiond